src/SEnergyCorrelator.ana.h

```diff
@@ -103,6 +115,7 @@ namespace SColdQcdCorrelatorAnalysis {
       if (m_config.doPackageCalc) {
         DoLocalCalcWithPackage( m_input.jets[iJet].GetPT()  );
       }
+      if(m_config.manualTwoPoint || m_config.manualThreePoint) DoLocalCalcManual(iJet, m_jetCstVector);
```

Comment:
Following above suggestions. The suggested bracketing/use of whitespace is just for consistency.

```suggestion
      if (m_config.doManualCalc) {
        DoLocalCalcManual( iJet, m_jetCstVector );
      }
```